src/repositories/DocumentRepository.ts

```diff
@@ -1,16 +1,16 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { HttpClient } from '../utilities/httpClient';
-import type { DocumentApi, FrontifyDocument } from '../types';
+import type { Document, DocumentApi } from '../types';
 
-export const getDocumentsWithoutDocumentGroupByProjectId = async (projectId: number): Promise<FrontifyDocument[]> => {
+export const getDocumentsWithoutDocumentGroupByProjectId = async (projectId: number): Promise<Document[]> => {
     const { result } = await HttpClient.get<DocumentApi[]>(
         `/api/document?project_id=${projectId}&exclude_grouped_documents`,
     );
     return result.data.map(mapDocumentApiToFrontifyDocument);
 };
 
-export const mapDocumentApiToFrontifyDocument = (documentApi: DocumentApi): FrontifyDocument => {
+export const mapDocumentApiToFrontifyDocument = (documentApi: DocumentApi): Document => {
```

Comment:
Should update method name as well:
```suggestion
export const mapDocumentApiToDocument = (documentApi: DocumentApi): Document => {
```